XenAdmin.sln

```diff
@@ -13,9 +13,6 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xva_verify", "xva_verify\xva_verify.csproj", "{2A70D7E7-EAB2-4C36-B3F4-85B79D2384B5}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "splash", "splash\splash.vcxproj", "{AFB19C9D-DD63-478B-A4A3-8452CBD0B9AB}"
-	ProjectSection(ProjectDependencies) = postProject

```

Comment:
I've also noticed this bit has been removed.

Reply:
What was this bit doing before it was removed? Ensuring that another project identified by this GUID is compiles after the splash project perhaps?

Reply:
I did some more googling, it seems this was needed previously to keep track of dependencies and after the upgrade it's moved to the proejcte references - which explains now the change in splash.vcxproj. Ok, my bad, let's leave this here and revert 6af64589103676ed57380eb3b2c2261a375aa779.